<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWidget</class>
 <widget class="qMRMLWidget" name="RegisterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleGroupBox" name="RegisterCollapsibleGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>A) Register volumes</string>
     </property>
     <property name="collapsedHeight">
      <number>5</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="RegisterGoToModulePushButton">
          <property name="toolTip">
           <string>Open the &quot;BrainsFit&quot; module</string>
          </property>
          <property name="text">
           <string>Go to Registration module</string>
          </property>
          <property name="workflow" stdset="0">
           <stringlist>
            <string>2</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RegisterApplyPushButton">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Register image together&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RegisterSavePushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save the output image</string>
          </property>
          <property name="text">
           <string>Save result </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="RegisterOutputNodeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Choose the &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Output skin model&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Output image volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
          <string></string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="RegisterFixedNodeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="RegisterFixedNodeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fixed image volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="RegisterMovingNodeComboBox">
        <property name="toolTip">
         <string>Select moving image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Moving</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="RegisterMovingNodeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Moving image volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="RegisterTypeLabel">
        <property name="text">
         <string>Registration type</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="RegisterTypeComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
        <item>
         <property name="text">
          <string>Initial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rigid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Affine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BSpline</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="RegisterOutputNodeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the output image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="baseName">
         <string/>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>RegisterWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>RegisterFixedNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RegisterWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>RegisterMovingNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RegisterWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>RegisterOutputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
